src/mlpack/bindings/cli/cli_option.hpp

```diff
@@ -131,59 +131,59 @@ class CLIOption
       #undef BASH_RED
       #undef BASH_CLEAR
 
-      outstr << "Parameter --" << boostName << " (" << data.alias << ") "
+      outstr << "Parameter --" << cliName << " (" << data.alias << ") "
              << "is defined multiple times with the same identifiers."
              << std::endl;
     }
 
-    CLI::Add(std::move(data));
+    CMD::Add(std::move(data));
```

Comment:
Now it is clear that we do need to rename `mlpack::core::CLI`, but I'm not sure if `CMD` is the right name to use, since it is used for all binding types.  Maybe a better name would be `IO`?  (Amusingly, that was the original name of the `CLI` back in ~2011-2013.)

Reply:
Agree `IO` seems to be a better option.
Hopping not create a bigger mess with regexp :+1: